utils/githooks/find_base.sh

```diff
@@ -23,9 +30,26 @@ if command -v gh > /dev/null 2>&1; then
         fi
     fi
 else
-    # With no 'gh' command installed then check against origin/master.
+    # With no 'gh' command installed, use the "closest" branch as the target,
+    # calculated as the sum of the commits this branch is ahead and behind.
+    # check master, then current release branches, then current feature branches.
     # shellcheck disable=SC2034
-    TARGET="$ORIGIN"/master
+    all_bases=("master")
+    all_bases+=($(git branch --list -r "$ORIGIN/release/*" \
```

Comment:
(lint) Prefer mapfile or read -a to split command output (or quote to avoid splitting). [SC2207]